### PR TITLE
Fix election id generation on the web front-end

### PR DESCRIPTION
**Pull request by GabrielFleischer**
The web front-end generates election ids based on the creation time of the LAO instead of the election's.

### CODE REVIEW

File: fe1-web/network/MessageApi.ts [hunk 1]
```diff
@@ -243,7 +243,7 @@ export function requestCreateElection(
   const message = new SetupElection({
     lao: currentLao.id,
     id: Hash.fromStringArray(
-      EventTags.ELECTION, currentLao.id.toString(), currentLao.creation.toString(), name,
+      EventTags.ELECTION, currentLao.id.toString(), time.toString(), name,
```
Comment (jbsv) [inline fe1-web/network/MessageApi.ts:246]: Was that a confusion between current Lao time and election creation time ?

Would it be possible to catch that in a test ?

**Comment by jbsv** [comment]
LGTM, but as it is fe1, I'll let @pierluca have an eye on it :-)

File: fe1-web/network/MessageApi.ts (hunk 1 above)
Comment (GabrielFleischer) [reply fe1-web/network/MessageApi.ts:246]: I am not very proficient with front-end tests. But it should be yes.

**Comment by GabrielFleischer** [comment]
I had not changed the id verification, it is now done too.

**Review by pierluca — APPROVED** [review]

**Pull request merged**